src/job/jobHandler.ts

```diff
@@ -554,8 +647,20 @@ export abstract class JobHandler {
       this._currJob._id,
       `* Starting Job with ID: ${this._currJob._id} and type: ${this._currJob.payload.jobType}`
     );
+    await this._logger.save(this._currJob._id, `* Starting Job with repo name: ${this._currJob.payload.repoName}`);
     try {
-      await this.build();
+      // TODO: MONOREPO feature flag needed here
+      if (
+        // process.env.FEATURE_FLAG_MONOREPO_PATH === 'true' &&
```

Comment:
Do we still want to add in the feature flag here?

Reply:
As of now, yes!